tests/e2e/blocks/audio-video.spec.js

```diff
@@ -0,0 +1,55 @@
+import {test, expect} from '../tools/lib/test-utils.js';
+import {publishPostAndVisit, createPostWithFeaturedImage} from '../tools/lib/post.js';
+import {searchAndInsertBlock} from '../tools/lib/editor.js';
+
+const YOUTUBE_TEST = 'https://www.youtube.com/watch?v=-CwkccAgKrs';
+const VIMEO_TEST = 'https://vimeo.com/120680405';
+const MP4_TEST = 'https://www.greenpeace.org/static/planet4-assets/tests/edge_of_the_wrold.mp4';
+
+const SOUNDCLOUD_TEST = 'https://soundcloud.com/greenpeaceuk-1/04-and-we-will-defend-requiem';
+const MP3_TEST = 'https://www.greenpeace.org/static/planet4-assets/tests/wochenserie_greenpeace.mp3';
+
+/**
+ * Add a Video or Audio block to a page with a specific link.
+ *
+ * @param {Object} page - The page object for interacting with the browser.
+ * @param {string} mediaType - The type of media added (Audio or Video).
+ * @param {string} mediaLink - The media file link (can be YouTube, Vimeo, mp4, mp3, SoundCloud).
+ */
+const addVideoOrAudioBlock = async ({page}, mediaType, mediaLink) => {
+  await searchAndInsertBlock({page}, mediaType, mediaType);
+  // We should close the sidebar before editing the block.
+  const toggleSidebar = await page.getByRole('button', {name: 'Toggle block inserter'});
```

Comment:
We need to close the sidebar before editing the block and adding a new one, because otherwise it stays open. I'm wondering if that's something that we should include directly in the searchAndInsertBlock function 🤔